### ISSUE TITLE
Consider adding an option to specify a custom path for e.g. `/bower_components`

### ISSUE THREAD

**Issue by eriklindebratt** [doberman/generator-doberman#4]
Since Bower has the option to specify a custom path for its components (instead of the default `/bower_components`), it would be sweet if you could set your own in e.g. a shell prompt.

**Comment by adriantomic** [comment]
@eriklindebratt Why not add a [.bowerrc](http://bower.io/docs/config/) file?

**Comment by eriklindebratt** [comment]
@adriantomic Definitely, but would be nice if the generator could prompt you to set a custom `directory` value in the generated `.bowerrc` file.
